components/Navigation.js

```diff
@@ -2,16 +2,23 @@ import Link from "next/link";
 import styled from "styled-components";
 import Flower from "@/public/flower.svg";
 import House from "@/public/house.svg";
+import AddPlantIcon from "@/public/badge-plus.svg";
 
 export default function Navigation() {
   return (
     <StyledNav>
-      <Link href="/" aria-label="Link to homepage">
+      <StyledLink href="/" aria-label="Link to homepage">
         <House />
-      </Link>
-      <Link href="/my-plants" aria-label="Link to my plants page">
+      </StyledLink>
+      <StyledAddLink
```

Comment:
You should implement some highlighting for the page the user is on. You can use the router for that purpose e.g. $active={router.path==="create"}